server/tenantConfig.js

```diff
@@ -38,7 +38,7 @@ async function layoutConfig() {
         maxSearchResults: 500,
         maxUploadSize: process.env.MAX_UPLOAD_SIZE || 10485760,
         autoCreateAndAllocateEnabled: configuration.autoCreateAndAllocateEnabled,
-        defaultTimeoutSeconds: isNaN(defaultTimeoutSeconds) ? 300 : defaultTimeoutSeconds,
+        defaultTimeoutSeconds: isNaN(defaultTimeoutSeconds) ? 1200 : defaultTimeoutSeconds,
```

Comment:
Is this ok to go in?

Reply:
Yeah - not really related to this PR but it just helps provide a more pleasant dev experience when there are not any session cookies